.github/bin/dist-version.ps1

```diff
@@ -19,8 +19,18 @@ New-Item -ItemType directory -Path obj -ErrorAction SilentlyContinue
 Write-Output $VersionPrefix
 Write-Output $VersionPrefix > obj/version_prefix.txt
 
-$HeadCommit = `
-  (Get-Content ".git/$((Get-Content .git/HEAD).Split()[1])").Substring(0, 7)
+$Event = Get-Content $env:GITHUB_EVENT_PATH | ConvertFrom-Json
+if ($Event.head_commit.id -ne $null) {
+  $HeadCommit = $Event.head_commit.id
+}
+if ($HeadCommit -eq $null) {
+  $HeadCommit = Get-Content .git/HEAD
```

Comment:
it's a bit confusing because `$HeadCommit` isn't a commit hash yet.

Reply:
It's actually a *commit*, it could be not a *commit hash* yet though…

Reply:
Of course it is. but I think representing *commit* and *commit hash* in same variable (`$HeadCommit`) is a bit confusing.

Reply:
Fixed! 5b4b5125b9776b4c3bad4817f32fd278147f1f67